hmvec/fft.py

```diff
@@ -52,7 +52,7 @@ def fft_integral(x,y,axis=-1):
 def analytic_fft_integral(ks): return np.sqrt(np.pi/2.)*np.exp(-ks**2./2.)*ks
 
 
-def generic_profile_fft(rhofunc_x,cmaxs,rss,zs,ks,xmax,nxs):
+def generic_profile_fft(rhofunc_x,cmaxs,rss,zs,ks,xmax,nxs,doMassNorm=True):
```

Comment:
Super minor comment -- don't use CamelCase for Python variables. do_mass_norm will be more consistent with the rest of this repo and also with the majority of Python codes that try to adhere to style guidelines.

Reply:
Great thanks for letting me know. I've renamed the variables.